### PR TITLE
Feature/to json

### PR DESCRIPTION
**Pull request by jlconlin**
Adding capability to convert RECONR to JSON representation.

### CODE REVIEW

File: src/lipservice/src/RECONR.hpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+           { "errint", card4.errint.value }
+  };
+}
+
+inline void to_json( nlohmann::json& JSON, const RECONR::Card5& card5 ) {
+  JSON = card5.cards.value;
+}
+
+inline void to_json( nlohmann::json& JSON, const RECONR::Card6& card6 ) {
+  JSON = { { "enode", card6.enode.value } };
```
Comment (whaeck) [inline src/lipservice/src/RECONR.hpp]: what is the actual name that the NJOY manual gives these values? I would use that instead. You do it everywhere else

Comment (whaeck) [reply src/lipservice/src/RECONR.hpp]: Nevermind.

File: src/lipservice/RECONR/test/RECONR.test.cpp [hunk 2]
```diff
@@ -10,146 +10,355 @@ std::string card1(" 21 22\n");
...
+                       "This a sample description TEXT Record for Card5" ],
+            "enode": [ ]
+          },
+          {
+            "mat": 1316, "ncards": 1, "ngrid": 0,
+            "err": 0.067, "tempr": 3.0, "errmax": 2.1, "errint": 8E-7,
+            "cards": [ "Another sample for Card5" ],
+            "enode": [ ]
+          }
+        ]
```
Comment (whaeck) [inline src/lipservice/RECONR/test/RECONR.test.cpp]: This is a lot nicer!

Comment (jlconlin) [reply src/lipservice/RECONR/test/RECONR.test.cpp]: I agree it is nicer.

**Pull request merged**